src/main/java/com/entropy/entropay/auth/service/AuthenticationService.java

```diff
@@ -0,0 +1,5 @@
+package com.entropy.entropay.auth.service;
+
+public interface AuthenticationService {
```

Comment:
do we actually need the interface? I don't think that we are going to have more than one implementation